distribution/ddf-common/src/main/resources/etc/org.apache.karaf.features.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<!--
+    This file was manually created for right now.
+-->
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0"
+                    xmlns:f="http://karaf.apache.org/xmlns/features/v1.5.0">
+
+    <!-- A list of blacklisted features XML repository URIs - they can't be added later -->
+    <blacklistedRepositories/>
+
+    <!-- A list of blacklisted feature identifiers that can't be installed in Karaf and are not part of the distribution -->
+    <blacklistedFeatures>
+        <feature>pax-jetty-http2</feature>
+        <feature version="[9.3,9.4.10]">pax-jetty</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-jetty</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http</feature>
+        <feature version="[7.0.0,7.2.2]">pax-war</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-tomcat</feature>
+        <feature version="[7.0.0,7.2.2]">pax-war-tomcat</feature>
+        <feature version="[7.0.0,7.2.2]">pax-jsf-support</feature>
+        <feature version="[7.0.0,7.2.2]">pax-jsf-resources-support</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-undertow</feature>
+        <feature>decanter-appender-cassandra</feature>
+        <feature>decanter-appender-cassandra-core</feature>
+    </blacklistedFeatures>
+
+    <!-- A list of blacklisted bundle URIs that are not installed even if they are part of some features -->
+    <blacklistedBundles/>
+
+    <!-- A list of repository URIs, feature identifiers and bundle URIs to override "dependency" flag -->
+    <overrideBundleDependency/>
+
+    <!-- A list of bundle URI replacements that allows changing external feature definitions -->
+    <bundleReplacements>
+        <bundle originalUri="mvn:org.apache.felix/org.apache.felix.scr/2.1.2"
+                replacement="mvn:org.apache.felix/org.apache.felix.scr/2.0.14"
+                mode="maven"/>
+    </bundleReplacements>
```

Comment:
This file was copied from https://github.com/codice/ddf/pull/3783/files#diff-b7fbb241181c3f9241f74dc815e2cf87, with the exception of this bundle replacement.